docs/release_notes/2024/sept2024.md

```diff
@@ -9,6 +9,27 @@ tags:
   - september
 ---
 
+## 3.3.11.\* (September 20, 2024)
+
+- Prophecy Python libs version: 1.9.14
+- Prophecy Scala libs version: 8.1.4
+
+### Features {#Features3311}
```

Comment:
@alexanderahn Jinja Syntax feature isn't there ???

Reply:
Does the item on line 21 not refer to it?